src/main/java/lotto/Lotto.java

```diff
@@ -1,20 +1,108 @@
 package lotto;
 
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import lotto.domain.LottoPolicy;
 
 public class Lotto {
+    private static final String ERROR_NOT_NUMERIC = "[ERROR] 로또 번호는 숫자만 입력 가능합니다.";
+    private static final String ERROR_BLANK = "[ERROR] 당첨 번호는 공백일 수 없습니다.";
+    private static final String ERROR_RANGE = "[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다.";
+    private static final String ERROR_DUPLICATE = "[ERROR] 로또 번호는 중복될 수 없습니다.";
+    private static final String ERROR_COUNT = "[ERROR] 로또 번호는 6개여야 합니다.";
+    private static final String NUMBERS_DELIMITER = ",";
     private final List<Integer> numbers;
 
     public Lotto(List<Integer> numbers) {
         validate(numbers);
         this.numbers = numbers;
     }
 
+    public static Lotto from(String userInput) {
+        validateUserInput(userInput);
+
+        List<Integer> numbers = parseInputToNumbers(userInput);
+
+        return new Lotto(numbers);
+    }
+
+    private static void validateUserInput(String input) {
+        if (input == null || input.isBlank()) {
+            throw new IllegalArgumentException(ERROR_BLANK);
+        }
+    }
+
+    private static List<Integer> parseInputToNumbers(String input) {
+        String[] numberStrings = input.split(NUMBERS_DELIMITER);
+
+        try {
+            return Arrays.stream(numberStrings)
+                    .map(String::trim) // " 1" -> "1" (공백 제거)
+                    .map(Integer::parseInt) // "1" -> 1
+                    .toList();
+        } catch (NumberFormatException e) {
+            // "1,a,3" 또는 "1,,3" 같은 오류를 여기서 잡습니다.
+            throw new IllegalArgumentException(ERROR_NOT_NUMERIC);
+        }
+    }
+
     private void validate(List<Integer> numbers) {
-        if (numbers.size() != 6) {
-            throw new IllegalArgumentException("[ERROR] 로또 번호는 6개여야 합니다.");
+        validateNumberCount(numbers);
+        validateNumberRange(numbers);
+        validateDuplicateNumber(numbers);
+    }
+
+    private void validateNumberRange(List<Integer> numbers) {
+        if (hasOutOfRange(numbers)) {
+            throw new IllegalArgumentException(ERROR_RANGE);
+        }
+    }
+
+    private boolean hasOutOfRange(List<Integer> numbers) {
+        for (Integer number : numbers) {
+            if (number < LottoPolicy.MIN_NUMBER || LottoPolicy.MAX_NUMBER < number) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
이 부분을 스트림을 사용하시면 아래와 같이 간결하게 표현하실 수 있습니다! 컬렉션을 다룰 때 스트림 API도 사용해보시는 걸 추천드려요.
```suggestion
    private boolean hasOutOfRange(List<Integer> numbers) {
        return numbers.stream()
                .anyMatch(number -> number < LottoPolicy.MIN_NUMBER || LottoPolicy.MAX_NUMBER < number);
    }
```